src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.js

```diff
@@ -56,7 +56,7 @@ class DeleteConfirmation extends React.Component<Props> {
                 <div className="bcs-DeleteConfirmation-promptMessage">
                     <FormattedMessage {...message} />
                 </div>
-                <div>
+                <div className="bcs-DeleteConfirmation-btn-container">
```

Comment:
Instead of `*-btn-container` can we name it `bcs-DeleteConfirmation-actions`?

Reply:
Updated! :)